recipe/recipe.yaml

```
@@ -0,0 +1,228 @@
+
+schema_version: 1
+
+context:
+  version: 2.3.1
+  # -1 means "not a dev version", a number > -1 will emit postfix "rc${{ rc_num }}"
```

comment is out-dated
```suggestion
  # "-1" means "not a dev version", a number != "-1" will emit postfix "rc${{ rc_num }}"
```